api/notd/manager.py

```diff
@@ -220,7 +221,8 @@ async def get_collection_daily_activities(self, address: str) -> List[Collection
                         minimumValue = min(minimumValue, collectionActivity.minimumValue) if minimumValue > 0 else collectionActivity.minimumValue
                         maximumValue = max(maximumValue, collectionActivity.maximumValue)
                     transferCount += collectionActivity.transferCount
-            collectionActivitiesPerDay.append(CollectionDailyActivity(date=currentDate,transferCount=transferCount,saleCount=saleCount,totalValue=totalValue,minimumValue=minimumValue,maximumValue=maximumValue,averageValue=0))
+                    averageValue += totalValue/transferCount
```

Comment:
spaces between math operations

Reply:
should this be saleCount?

Reply:
also, shouldnt this be tabbed all the way back (and you can remove the averageValue=0 at the top cos its not something we add to constantly, it only needs to be set once at the end